sbin/submitty_daemon_jobs/submitty_jobs/jobs.py

```diff
@@ -355,3 +355,24 @@ def run_job(self):
             subprocess.run(["sudo", "/usr/local/submitty/sbin/adduser_course.py", head_instructor, semester, course], stdout=output_file, stderr=output_file)
             if VERIFIED_ADMIN_USER != "":
                 subprocess.run(["sudo", "/usr/local/submitty/sbin/adduser_course.py", VERIFIED_ADMIN_USER, semester, course], stdout=output_file, stderr=output_file)
+
+
+class UpdateDockerImages(AbstractJob):
+    def run_job(self):
+        today = datetime.datetime.now()
+        log_path = os.path.join(DATA_DIR, "logs", "docker")
+        try:
```

Comment:
Should this job trying to make the `logs` dir, I think most jobs assume they get created ahead of time in the install scripts